plugin/src/main.rs

```diff
@@ -0,0 +1,91 @@
+use clap::Parser;
+use plugin::ExecuteOperation;
+use kubectl_plugin::resources;
+use std::{env, ops::Deref};
+
+pub(crate) mod cli_utils;
+
+
+#[derive(Parser, Debug)]
+#[clap(name = utils::package_description!(), version = utils::version_info_str!())]
+#[group(skip)]
+pub struct CliArgs {
```

Comment:
```suggestion
pub(crate) struct CliArgs {
```

Reply:
Tried dont need (crate) also :) . Im exposing all under `cli_utils` module as pub for puls8 plugin.